terraform/modules/services/api_pvsite/docker-compose.yml

```diff
@@ -9,6 +9,7 @@ services:
       DB_URL_PV: $DB_URL_PV
       ORIGINS: $ORIGINS
       FAKE: $FAKE
+      SENTRY_DNS: $SENTRY_DNS
```

Comment:
whats this?

Reply:
Brad wants to run the API logs through Sentry - it's what they use to alert on failures for the frontend.